python/activator/middleware_interface.py

```diff
@@ -231,8 +282,11 @@ def prep_butler(self, visit: Visit) -> None:
                                 directory=self.central_butler.datastore.root,
                                 transfer="copy")
 
-        self._prep_collections()
-        self._prep_pipeline(visit)
+        # Create unique run to store this visit's raws. This makes it easier to
```

Comment:
Just to be clear, are you recommending that we not sync over the raws even before taking on DM-36051?

The problem I'm concerned about is that `upload.py` currently recycles the same four visit/exposure IDs over and over again. I don't see how DM-36051 solves it, since it would also need to resolve duplicate raws.

Reply:
I left the custom raw collections in place, but added a TODO conditional on DM-36051 or resolving the `upload.py` exposure ID problem.